src/_api/device_directory.ts

```diff
@@ -325,7 +325,7 @@ export interface DeviceDataPutRequest {
      */
     "custom_attributes"?: { [key: string]: string; };
     /**
-     * The fingerprint of the device certificate.
+     * This is not part of the certificate but rather the SH-256 hash of the certificate.
```

Comment:
This file is autogenerated so you don't need to make any changes here as they will be overwritten next time a swagger change is merged.